### PR TITLE
Cleanup2

### PR DESCRIPTION
**Pull request by bluzarraga**

### CODE REVIEW

File: helm-charts/helm-api/ibm_cloud_pak/manifest.yaml [hunk 1]
```diff
@@ -1,111 +1,142 @@
...
+    platform:
+      os: linux
+      arch: amd64
+  - repository: icp-helm-api@sha256:8551180b18e4273775e258ebf3680ab7096ddd68d5d6e01b65e5c3c9661fffb4
+    pull-repository: quay.io/opencloudio/icp-helm-api@sha256:8551180b18e4273775e258ebf3680ab7096ddd68d5d6e01b65e5c3c9661fffb4
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository: icp-helm-api@sha256:8551180b18e4273775e258ebf3680ab7096ddd68d5d6e01b65e5c3c9661fffb4
+    pull-repository: quay.io/opencloudio/icp-helm-api@sha256:8551180b18e4273775e258ebf3680ab7096ddd68d5d6e01b65e5c3c9661fffb4
```
Comment (bitscuit) [inline helm-charts/helm-api/ibm_cloud_pak/manifest.yaml]: icp-helm-api is at 3.4.1 and digest is `e73ac971497212eb3b8904918aa975136c8e8e8c47a520273e51e65fd9bff214` now

File: helm-charts/helm-api/ibm_cloud_pak/manifest.yaml [hunk 2]
```diff
@@ -1,111 +1,142 @@
...
+    platform:
+      os: linux
+      arch: amd64
+  - repository : icp-helm-rudder@sha256:5eba2b4052eb80a7ed1e0a91f3e0bcd1e4ae8412e604cfd4dc990daf2e9da9c7
+    pull-repository: quay.io/opencloudio/icp-helm-rudder@sha256:5eba2b4052eb80a7ed1e0a91f3e0bcd1e4ae8412e604cfd4dc990daf2e9da9c7
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository : icp-helm-rudder@sha256:5eba2b4052eb80a7ed1e0a91f3e0bcd1e4ae8412e604cfd4dc990daf2e9da9c7
+    pull-repository: quay.io/opencloudio/icp-helm-rudder@sha256:5eba2b4052eb80a7ed1e0a91f3e0bcd1e4ae8412e604cfd4dc990daf2e9da9c7
```
Comment (bitscuit) [inline helm-charts/helm-api/ibm_cloud_pak/manifest.yaml]: icp-helm-rudder is at 3.3.1 and digest is `437c980277a700f7f53db1ac88ae1d10488033b52422bf17654a82717c6f22af` now

File: helm-charts/helm-api/ibm_cloud_pak/manifest.yaml [hunk 3]
```diff
@@ -1,111 +1,142 @@
...
+    platform:
+      os: linux
+      arch: amd64
+  - repository : tiller@sha256:bce0b2a08c309e41f08b5026d1a3904c271b09a2071d0380b90b47aae050cd7f
+    pull-repository: quay.io/opencloudio/tiller@sha256:bce0b2a08c309e41f08b5026d1a3904c271b09a2071d0380b90b47aae050cd7f
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository : tiller@sha256:bce0b2a08c309e41f08b5026d1a3904c271b09a2071d0380b90b47aae050cd7f
+    pull-repository: quay.io/opencloudio/tiller@sha256:bce0b2a08c309e41f08b5026d1a3904c271b09a2071d0380b90b47aae050cd7f
```
Comment (bitscuit) [inline helm-charts/helm-api/ibm_cloud_pak/manifest.yaml]: tiller is at v2.16.1-cs-3.3.1 and digest is `a7539700cdc49623efd9c049760d9b5c32329c498be55f30a87835afdc43fe47`

**Review by bitscuit — CHANGES_REQUESTED** [review]

**Comment by bitscuit** [comment]
Couple more files that are using old digest values

https://github.com/IBM/ibm-helm-api-operator/blob/cleanup2/deploy/operator.yaml#L60-L65
https://github.com/IBM/ibm-helm-api-operator/blob/cleanup2/deploy/olm-catalog/ibm-helm-api-operator/3.6.0/ibm-helm-api-operator.v3.6.0.clusterserviceversion.yaml#L260-L265

File: helm-charts/helm-api/ibm_cloud_pak/manifest.yaml [hunk 4]
```diff
@@ -1,111 +1,142 @@
...
-  - repository : icp-helm-rudder-s390x:3.2.2
-    pull-repository: hyc-cloud-private-stable-docker-local.artifactory.swg-devops.com/ibmcom-s390x/icp-helm-rudder:3.2.2
-    pull-authorization:
-      username:
-        env: ARTIFACTORY_USERNAME
-      password:
-        env: ARTIFACTORY_APIKEY
+      arch: ppc64le
+  - repository : tiller@sha256:a7539700cdc49623efd9c049760d9b5c32329c498be55f30a87835afdc43fe47
+    pull-repository: quay.io/opencloudio/tiller@sha256:a7539700cdc49623efd9c049760d9b5c32329c498be55f30a87835afdc43fe47
```
Comment (cheewaio) [inline helm-charts/helm-api/ibm_cloud_pak/manifest.yaml:69]: should the digest be arch specific?

**Review by cheewaio — CHANGES_REQUESTED** [review]
to update operand image digest, there are a few places you will need to update
- operator.yaml
- csv
- values.yaml
- manifest.yaml

File: helm-charts/helm-api/ibm_cloud_pak/manifest.yaml (hunk 4 above)
Comment (bluzarraga) [reply helm-charts/helm-api/ibm_cloud_pak/manifest.yaml:69]: The bundle likes it that way

**Review by cheewaio — APPROVED** [review]
/lgtm

**Pull request merged**